fabric-biome-api-v1/src/main/java/net/fabricmc/fabric/api/biome/v1/BiomeSelectionContext.java

```diff
@@ -115,4 +115,16 @@ default boolean hasPlacedFeature(RegistryKey<PlacedFeature> key) {
 	 * {@return true if this biome is in the given {@link TagKey}}.
 	 */
 	boolean hasTag(TagKey<Biome> tag);
+
+	/**
+	 * {@return true if this biome is in any of the given {@link TagKey}s}.
+	 */
+	default boolean hasTag(TagKey<Biome>... tags) {
```

Comment:
It doesn't sound obvious. I think we need to add `anyTagMatch`, `allTagMatch` and `noneTagMatch`

Reply:
I agree I think something like this would be ideal here

Reply:
To be clear: I'm suggesting the same thing as `Stream::anyMatch`, `Stream::allMatch`, `Stream::noneMatch`